Cloud Application Manager/Administering Your Organization/resource-tags.md

```diff
@@ -45,58 +44,64 @@ All users with Cloud Application Manager organization administrator access.
 | Instance ID | ID assigned by Cloud Application Manager, for example, i-extwmf. |
 | Provider name | Provider defined in Cloud Application Manager to which the instance was deployed. |
 | Service instance ID | Unique ID for every machine created for the instance, for example eb-ek73d-1, eb-ek73d-2. |
-| Service ID | ID for the type of service deployed from the provider, for example eb-ek73d. |
 | User email | Email of the user who deployed the instance. |
+| Service ID | ID for the type of service deployed from the provider, for example eb-ek73d. |
+| Support ID | This ID relates an instance in case you ask for support regarding it. There might be more than one ID in case the instance is associated with more than one machine. |
 | User ID | A unique ID to identify the Cloud Application Manager user. |
 | Workspace ID | Unique workspace ID where the instance was deployed. |
 | Workspace name | Name of the workspace the instance was deployed from. |
 
-### Applying Tags for Your Organization
+### Applying Tags for your Organization
+
 Only Cloud Application Manager users in the administrator role can apply tags. Follow these steps to apply one.
 
 **Steps**
+
 1. [Log in to Cloud Application Manager.](//www.ctl.io/cloud-application-manager/)
 
 2. From the menu drop-down on the top right, select **Organizations > Your Org Name**
 
 3. Click **Settings and scroll to the Tags section**.
 
 4. Click **Add** and enter a key and value for the tag.
-   * To enter a preset value, click the Custom Value drop-down to select one.
+   * To enter a preset value, click the Value drop-down to select one.
    * To enter a custom value, simply type in the Custom Value field.
    * **Note:** The maximum length is 125 characters for the key and 250 characters for the value. Tags that contain unicode non-ASCII characters (ex: +=\*&!@#) are ignored. Such tags are not applied to the instance in Google Cloud and OpenStack.
 
 5. When done, click **Save** to add the tag.
 
-**Note:** At this time, you can’t modify an existing tag, but you can remove and create another in its place. To remove a tag, go the Settings and under Tags, click (x) and the tick mark against it.
+**Note:** Edition of tags is not currently supported. But you can remove a tag and create another in its place. To delete a tag, go to **Settings** and under Tags, click the X button next to the name of the tag you want to remove.
 
 ### Reporting on Cloud Application Manager Tags
-One of the chief benefits of tagging is that you can report and analyze how Cloud Application Manager resources are consumed throughout your organization. Currently, the reporting capabilities depend on what your cloud provider natively supports.
+
+One of the main benefits of tagging is that you can report and analyze how Cloud Application Manager resources are consumed throughout your organization. Currently, the reporting capabilities depend on what your cloud provider natively supports.
```

Comment:
I would add here a link to our Analytics section that can also benefit from tagging